test/integration/testdata/check.golden

```diff
@@ -35,10 +35,37 @@ linkerd-identity
 √ trust anchors are using supported crypto algorithm
 √ trust anchors are within their validity period
 √ trust anchors are valid for at least 60 days
-√ issuer cert is using supported crypto algorithm
-√ issuer cert is within its validity period
-√ issuer cert is valid for at least 60 days
-√ issuer cert is issued by the trust anchor
+√ cert is using supported crypto algorithm
+√ cert is within its validity period
+√ cert is valid for at least 60 days
+√ cert is issued by the trust anchor
+
+linkerd-tap-tls
```

Comment:
This feels a bit verbose for the successful case.  What about condensing this into a single check for each webhook that does all of the validation and returns the specific failure in the case of an error?  Something like this:

```
webhooks
----------
√ tap API server has valid cert
√ proxy-injector webhook has valid cert
√ sp-validator webhookd has valid cert
```

A side-benefit of this is that we can fetch and use the certificate within the same check which means we no longer need to store it on the health check struck.

Reply:
Can the tap TLS check be grouped under webhooks. Not too sure about that one

Reply:
`webhooks and apiservices` 🤣 

Not sure about the naming, but it makes conceptual sense to me to group these together.  They are each isolated trust chains with a caBundle given to the kubernetes API.